cypress/component/Dataset/datasetTerm.json

```
@@ -0,0 +1,48 @@
+{
+  "datasetId": 1975,
+  "name": "ExteranlAccessTestJL1",
+  "datasetName": "ExteranlAccessTestJL1",
```

For higher fidelity test data where these values match the corresponding value in study.studyName in line 34:

```suggestion
  "name": "ExternalAccessTestJL1",
  "datasetName": "ExternalAccessTestJL1",
```